wca-states.md

```diff
@@ -3,10 +3,10 @@
 <version>Version: June 19, 2017
 
 
-The WCA recognizes a total of 199 independent countries.
+The WCA recognizes a total of 199 countries/regions:
 
 
-## <states-list>UN Member States:
+## <states-list>
```

Comment:
Do we know how this will look? I'm concerned it will introduce a bit empty white line in the final page.

Reply:
Ah, I just saw https://github.com/thewca/worldcubeassociation.org/pull/3311/files. Please disregard this message.